src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -23,6 +23,7 @@
 class FileContextToken(Token):
     filename: str
 
+    # pylint: disable=signature-differs
```

Comment:
@mortalisk , can you verify this?

Reply:
What are you suggesting @mortalisk to verify? I could not understand. 
Would you like that this issue is reviewed by him for some reason or was only this comment?

Reply:
So, it's complaining that we changed the __Init__ signature as compared with the super-class? I think that is fine, don't get why it is complaining about that in the first place.

Reply:
Fine with me, @hnformentin  you may review the rest :)